client/src/components/App/Content/UsersManagement/UsersManagementStyles.ts

```diff
@@ -33,10 +33,10 @@ const useStyles = makeStyles((theme: Theme) => ({
     active: {},
     filters: {
         flip: false,
+        width: '90vw',
         margin: theme.spacing(2),
         display: 'flex',
-        justifyContent: 'flex-end',
-        marginLeft: '10vw'
+        justifyContent: 'space-between',
```

Comment:
you can use justify="space-between" the Grid instead, Display: flex is redundant and you can use spacing={2} on Grid as well

Reply:
meaning `<Grid justify="space-between" spacing={2} />`

Reply:
fixed